composable_prepare_nic_configs.yml

```diff
@@ -243,3 +263,60 @@
           machine_count: "{{ machine_count | combine({item.key: (item.value |int - ((item.key == pci_node_type)|ternary(pci_node_count, 0)))}) }}"
         with_dict: "{{ machine_count }}"
         when: passthrough_nvme is defined and pci_node_count is defined
+
+      - block:
+          - name: prepare network-environment_v2.yaml.j2
+            template:
+              src: network-environment_v2.yaml.j2
+              dest: "{{ ansible_user_dir }}/virt/network/network-environment_v2.yaml.j2"
+
+          - name: prepare network_data_v2.yaml.j2
+            template:
+              src: network_data_v2.yaml.j2
+              dest: "{{ ansible_user_dir }}/virt/network/network_data_v2.yaml.j2"
+
+          - name: prepare vip_data.yaml.j2
+            template:
+              src: vip_data.yaml.j2
+              dest: "{{ ansible_user_dir }}/virt/network/vip_data.yaml.j2"
+
+          - name: Create nic_config_path dir
+            file:
+              path: "~/virt/network/vlans/"
+              state: directory
+            delegate_to: undercloud
+
+          - name: prepare controller.j2.j2
+            vars:
+              ctlplane_interface: "{{ controller_ctlplane_interface | default(ctlplane_interfaces[0]) }}"
+              external_interface: "{{ controller_external_interface | default(external_interfaces[0]) }}"
+              tenant_interface: "{{ controller_tenant_interface | default(tenant_interfaces[0]) }}"
+              isolated_interface: "{{ controller_isolated_interface | default(isolated_interfaces[0]) }}"
+              openshift_vlan_interface: "{{ controller_tenant_interface | default(openshift_vlan_interfaces[0]) }}"
+            template:
+              src: controller.j2.j2
+              dest: "~/virt/network/vlans/controller.j2"
+              force: yes
+            delegate_to: undercloud
+
+          - name: prepare compute.j2.j2
+            vars:
+              ctlplane_interface: "{{ item.1 }}"
+              external_interface: "{{ item.2 }}"
+              tenant_interface: "{{ item.3 }}"
+              isolated_interface: "{{ item.4 }}"
+              openshift_vlan_interface: "{{ item.5 }}"
+            template:
+              src: compute.j2.j2
+              dest: "~/virt/network/vlans//compute_{{ item.0 }}.j2"
+              force: yes
+            delegate_to: undercloud
+            with_together:
+              - "{{ machine_types }}"
+              - "{{ ctlplane_interfaces }}"
+              - "{{ external_interfaces }}"
+              - "{{ tenant_interfaces }}"
+              - "{{ isolated_interfaces }}"
+              - "{{ openshift_vlan_interfaces }}"
+
+        when: osp_release|int == 17
```

Comment:
can you use >= 17, it won't work on the later versions. it will be good to change all the == 17 to >= 17

Reply:
Done

Reply:
still i am seeing ==